docs/swupd.1.rst

```diff
@@ -266,6 +266,39 @@ SUBCOMMANDS
         Omit checking hash values. Instead only corrects missing files
         and directories and/or symlinks.
 
+    - `-Y, --picky`
+
+        List (without --fix) or remove (with --fix) files which should
+        not exist. Only files listed in the manifests should exist.
+
+    - `-X, --picky-tree=[PATH]`
+
+        Selects the sub-tree where --picky looks for extra files. To be
+        specified as absolute path. The default is `/usr`.
+
+    - `-w, --picky-whitelist=[RE]`
+
+        Any path matching the POSIX extended regular expression is
+        ignored by --picky. Matched directories get skipped
+        completely. Directory paths end in a slash. This makes it
+        possible to write expressions that match a directory but not
+        something else that happens to have the same base name.
```

Comment:
Yes, because directories are special I wanted to allow matching them separately. I also wanted to ensure that a regex like ^/usr/local does not accidentally also matches /usr/local.old (the mistake made in the current code).

But I agree, the former is a rather weak use case and the later is better solved by anchoring with $.

I'll revise the patch so that anchoring is enforced and without the trailing slash special case.